### ISSUE TITLE
Uncaught AttributeError in sendchat.py line 30

### ISSUE THREAD

**Issue by jjaimealeman** [paul-gauthier/aider#1868]
Aider version: 0.55.0
Python version: 3.12.6
Platform: Linux-6.10.10-arch1-1-x86_64-with-glibc2.40
Python implementation: CPython
Virtual environment: Yes
OS: Linux 6.10.10-arch1-1 (64bit)
Git version: git version 2.46.1

An uncaught exception occurred:

```
Traceback (most recent call last):
  File "aider", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "main.py", line 694, in main
    " working dir."
    ^^^^^^^^^^^
  File "base_coder.py", line 730, in run
    self.run_one(user_message, preproc)
  File "base_coder.py", line 773, in run_one
    list(self.send_message(message))
  File "base_coder.py", line 1183, in send_message
    return
           
  File "base_coder.py", line 1863, in auto_commit
    try:
         
  File "repo.py", line 101, in commit
    return
           
  File "repo.py", line 186, in get_commit_message
    dict(role="user", content=content),
                     ^^^^^^^^^^^^^^^^^^
  File "sendchat.py", line 38, in wrapper
    retry_exceptions(),
    ^^^^^^^^^^^^^^^^^^
  File "sendchat.py", line 30, in retry_exceptions
    litellm.llms.anthropic.chat.AnthropicError,
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'litellm.llms.anthropic' has no attribute 'AnthropicError'

```

**Comment by jjaimealeman** [comment]
So this issue was created automagically, pretty fukken cool guys!

Not sure what additional information you may need from.

**Comment by fry69** [comment]
Thank you for filing this issue.

The fix is in the latest version (`0.58.1`), please update aider with
```
aider --upgrade
```
Please do not forget to exit and restart aider after it updated itself.

Duplicate #1278
